sympy/ntheory/tests/test_factor_.py

```diff
@@ -301,18 +301,6 @@ def test_factorint():
     assert str(factorint(Dict(n))) == sans
 
 
-def test_factorint_flint_int_issue():
-    """
-    Because of a bug in flint, we are temporarily wrapping int in `factorint`.
-    When this fails, the int wrapper is no longer needed in factorint.
-
-    https://github.com/flintlib/python-flint/issues/92
-    https://github.com/sympy/sympy/pull/25749
-    """
-    if GROUND_TYPES == 'flint':
-        assert pow(2, 5, SYMPY_INTS[1](1000)) == 1
```

Comment:
I don't see anything better than just deleting this. This test will pass for `python-flint < 0.5.0` but will fail for `flint >= 0.5.0`. Either way it does not test anything meaningful about SymPy's codebase. We probably do not want to add a Flint version check right now for wherever is the wrapping code. I guess maybe just adding a comment there is better but I don't know where that comment should go.

Reply:
The Flint jobs will fail on all PRs until this change is made. This change could be pulled out as a separate PR to stop the failures showing up on unrelated PRs.